clients/admin-ui/src/features/system/system-form-declaration-tab/PrivacyDeclarationForm.tsx

```diff
@@ -0,0 +1,408 @@
+          tooltip="How long is personal data retained for this purpose?"
+          variant="stacked"
+        />
+      </SystemFormInputGroup>
+      <SystemFormInputGroup heading="Features">
+        <CustomCreatableSelect
+          name="features"
+          label="Features"
+          tooltip="What are some features of how data is processed?"
+          variant="stacked"
+          options={[]}
+          disableMenu
+          isMulti
+        />
+      </SystemFormInputGroup>
+      <SystemFormInputGroup heading="Dataset reference">
+        <CustomSelect
+          name="dataset_references"
+          label="Dataset references"
+          options={datasetSelectOptions}
+          tooltip="Is there a dataset configured for this system?"
+          isMulti
+          variant="stacked"
+        />
+      </SystemFormInputGroup>
+      <SystemFormInputGroup heading="Special categories of processing">
+        <Stack spacing={0}>
+          <CustomSwitch
+            name="processes_special_category_data"
+            label="This system processes Special Category data"
+            tooltip="Is this system processing special category data as defined by GDPR Article 9?"
+            variant="stacked"
+          />
+          <Collapse
+            in={values.processes_special_category_data}
+            animateOpacity
+            style={{ overflow: "visible" }}
+          >
+            <Box mt={4}>
+              <CustomSelect
+                name="legal_basis_for_special_category_processing"
+                label="Legal basis for processing"
+                options={legalBasisForSpecialCategoryOptions}
+                isRequired={values.processes_special_category_data}
+                tooltip="What is the legal basis under which the special category data is processed?"
+                variant="stacked"
+              />
+            </Box>
+          </Collapse>
+        </Stack>
+      </SystemFormInputGroup>
+      <SystemFormInputGroup heading="Third parties">
+        <Stack spacing={0}>
+          <CustomSwitch
+            name="data_shared_with_third_parties"
+            label="This system shares data with 3rd parties for this purpose"
+            tooltip="Does this system disclose, sell, or share personal data collected for this business use with 3rd parties?"
+            variant="stacked"
+          />
+          <Collapse
+            in={values.data_shared_with_third_parties}
+            animateOpacity
+            style={{ overflow: "visible" }}
+          >
+            <Stack mt={4} spacing={4}>
+              <CustomTextInput
+                name="third_parties"
+                label="Third parties"
+                tooltip="Which type of third parties is the data shared with?"
+                variant="stacked"
+              />
+              <CustomSelect
+                name="shared_categories"
+                label="Shared categories"
+                options={allDataCategories.map((c) => ({
```

Comment:
I'm running into an issue when trying to save this field sometimes. Here is what the backend is returning

```json
{
    "detail": [
        {
            "loc": [
                "body",
                "privacy_declarations",
                0,
                "shared_categories"
            ],
            "msg": "value is not a valid list",
            "type": "type_error.list"
        }
    ]
}
```